fss/src/app/children/tabs/family-info-tab.component.ts

```diff
@@ -18,52 +18,35 @@ export class FamilyTabComponent implements OnInit, OnChanges {
     ) {}
 
     ngOnInit(): void {
+        this.restService.changeEmitter.subscribe(() => this.ngOnChanges({} as SimpleChanges));
```

Comment:
This was all that was needed to actually fix the bug... The rest is just unifying the child save button and some code cleanup.